src/Injector.tsx

```diff
@@ -69,7 +67,7 @@ export const Injector: React.FC<React.PropsWithChildren<Props>> = ({ store, chil
   const commandResolver = new CommandResolverImpl(new CommandManagerResolverImpl(store, apiHelper));
   const urlManager = new UrlManagerImpl(featureManager, baseUrl);
   const fileFetcher = new FileFetcherImpl(apiHelper);
-  const environmentModifier = EnvironmentModifierImpl();
+  const environmentModifier = useEnvironmentModifierImpl();
```

Comment:
We are keeping the name EnvironmentModifierImpl? Is there nothing less cryptic?

Reply:
it will be addressed in the graphQL iteration